spec/system/activity_notifications_spec.rb

```diff
@@ -165,7 +168,8 @@
 
       click_link('Zur Antwort', href: request_path(request, anchor: "message-#{reply_by_same_contributor.id}"))
       expect(page).to have_text("I'm a reply from #{contributor_two.name}")
-      click_link('nachfragen', href: contributor_request_path(contributor_two, request))
+      find("a[href='#{contributor_request_path(contributor_two, request)}']", text: 'nachfragen').trigger('click')
+
```

Comment:
was getting a `Exception: Capybara::Cuprite::MouseEventFailed` for overlapping elements. In this case, I think this is just because we are running in headless and I know that the navbar is not overlapping this link in a real browser. Using `trigger('click')` ignores potentially overlapping elements.